docs/_articles/en/lwc/linting.md

```diff
@@ -13,7 +13,7 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ### For a New Project
 
-If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
+If you create an SFDX project using the `sfdx project:generate` command, your project contains a `package.json` file with the ESLint plugin already included.
```

Comment:
@CristiCanizales should `sfdx project:generate` be `sf project:generate`?

Reply:
If you would prefer to attend to the migration using `sfdx` and then revisit with a separate work item to address command name changes, I am ok with that.